tests/functional/spec/features/section_summary/section_summary.spec.js

```diff
@@ -15,6 +15,12 @@ describe("Section Summary", () => {
       browser.openQuestionnaire("test_section_summary.json");
```

Comment:
Need to test scrolling behaviour after https://github.com/ONSdigital/eq-questionnaire-runner/pull/762